FlowCrypt/Functionality/Services/EKMService.swift

```diff
@@ -58,13 +56,11 @@ final class EKMService: EKMServiceType {
                 }
                 await viewController.showToast("refresh_key_success".localized)
             } catch {
+                // since this is an update function that happens on every startup
+                // it's ok if it's skipped sometimes - keys will be updated next time
                 if error is ApiError {
                     return
                 }
-                // Do not display error alert when no keys are returned
-                if let ekmError = error as? EmailKeyManagerApiError, ekmError == .noKeys {
-                    return
-                }
                 // Do not display error alert when user cancels pass phrase prompt
                 if let refreshError = error as? RefreshKeyError, refreshError == .cancelPassPhrase {
                     return
```

Comment:
I've unresolved a previous comment, since this one is still here but can be removed

Reply:
Yeah, I'm currently taking care of it.

Reply:
Yeah, I agree.
Addressed all review comments though.

Reply:
And ready to review